### PR TITLE
Update changelog with entries from 2.9.3 to current

### PR DESCRIPTION
**Pull request by mikehardy**
Had been doing these manually, forgetting/skipping the actual process
publish improvements to force the sync between AnkiDroid releases and
this project coming to avoid that...

### CODE REVIEW

**Comment by timrae** [comment]
@mikehardy
Have you been updating the local copy of the changelog? It's there to prevent the server from getting inundated during releases.

**Comment by timrae** [comment]
Totally forgot to tell you about it or document it anywhere, sorry!

**Comment by mikehardy** [comment]
@timrae yeah I had been hand-hacking it so the bundled version in the app was up to date but letting the ankidroiddocs version slide. Honestly I knew about it but was being lazy while we were iterating quickly. Now I've made the tools/release.sh script auto-render it for public releases (with an error if it can't for whatever reason - missing tool perhaps, or if the current version being built doesn't have an entry). That should make it the same time cost to just do it right so I've un-lazied myself :-)

File: .travis.yml [hunk 1]
```diff
@@ -4,4 +4,4 @@ install: gem install asciidoctor
 script: "./tools/convert.sh"
 notifications:
   email:
-    - alexander.grueneberg@googlemail.com
+    - github@mikehardy.net
```
Comment (agrueneberg) [inline .travis.yml:7]: @mikehardy, thanks for updating this. I can't remember ever receiving any emails for this pipeline except from forks that don't have permission to perform the update. There are not many, though.

**Comment by agrueneberg** [comment]
@mikehardy, so if I understand correctly you have now incorporated this pipeline into the release script so that the changelog gets automatically updated? Please let me know if there is something wrong with this pipeline, I may be able to still piece it together after all these years :)

**Comment by mikehardy** [comment]
Hey @agrueneberg ! What I did in Anki-Android/tools/release.sh was just make the assumption (which the script verifies and explains on error) that ankidroiddocs is checked out in parallel to Anki-Android, and it just attempts to use asciidoctor to render the changelog into the app's assets as you publish it. That's all local, but the effect (since tools/release.sh greps for the version you're about to publish and fails if it doesn't see it...) is that it makes sure that ankidroiddocs/changelog.asc has been updated. At that point it's a human calculus - anyone up against that error message trying to publish is going to see that updating ankidroiddocs/changelog.asc correctly is the path of least resistance so they'll do the needful

At which point your .travis.yml (which still works wonderfully, actually) will make sure it goes public

**Comment by agrueneberg** [comment]
Thanks for the details! And kudos to Travis for supporting this pipeline for so many years!

**Pull request merged**